File: Linguagem C/soma_impares.c
#include <stdio.h>
#include <stdlib.h>

int main() {

    int i, n1,n2, soma,x;

    printf("Digite dois numeros: \n");
    scanf("%d", &n1);
    scanf("%d", &n2);
    if (n1 > n2) {
        x = n1;
        n1 = n2;
        n2 = x;
    }

    soma = 0;
    for (i = n1+1; i < n2; i++) {
        if (i % 2 != 0) {
            soma = soma + i;
        }

    }
    printf("SOMA DOS IMPARES = %d", soma);




    return 0;
}

File: Linguagem Java/soma_impares.java
import java.util.Locale;
import java.util.Scanner;

public class soma_impares {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		System.out.println("Digite dois numeros: ");
		int x = sc.nextInt();
		int y = sc.nextInt();
		if (x > y) {
			int a = x;
			x = y;
			y = a;
		}
		
		int soma = 0;
		for(int i = x+1; i < y; i++) {
			if (i  % 2 != 0) {
				soma = soma + i;
			}
		}
		System.out.println("Soma dos impares = " + soma);
		
		sc.close();

	}

}
